src/commands/deploy/genezio.ts

```diff
@@ -675,7 +676,26 @@ export async function deployFrontend(
 
     try {
         await doAdaptiveLogAction(`Building frontend ${index + 1}`, async () => {
-            await runScript(frontend.scripts?.build, frontend.path);
+            debugLogger.debug(
+                `Running build script for frontend ${index + 1}`,
+                frontend.scripts?.build,
+            );
+
+            const projectEnvDetails = await getProjectEnvFromProjectByName(name, stage);
+            if (!projectEnvDetails) {
+                throw new UserError("Project environment not found.");
+            }
+
+            const functions = projectEnvDetails.functions?.map((f) => ({
+                name: f.name.replace(/-([a-z])/g, (g) => g[1].toUpperCase()) + "ApiUrl",
```

Comment:
Why to uppercase and not to lowercase? It means that I should write variables like this: `${{HELLOApiUrl}}`. You are also doing this in local.

Reply:
Nope, it won't be capitalized, it will be camelCase: `${{helloApiUrl}}`.

I extracted the logic in a utils method `kebabToCamelCase`. This should increase readability.